io.gitlab.persiangolf.voicegen.yml

```diff
@@ -0,0 +1,67 @@
+app-id: io.gitlab.persiangolf.voicegen
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: voicegen
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+
+
+modules:
+  #popt is a dependancy for svox pico
+  - name: popt
+    sources:
+      - type: archive
+        url: "http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.18.tar.gz"
+        sha256: 5159bc03a20b28ce363aa96765f37df99ea4d8850b1ece17d1e6ad5c24fdc5d1
+
+
+  - name: svox-pico
+    subdir: pico
+    buildsystem: autotools
+    sources:
+    - type: git
+      url: https://github.com/naggety/picotts.git
+      commit: 21089d223e177ba3cb7e385db8613a093dff74b5
+
+  - name: ffmpeg
+    config-opts:
+      - --enable-gpl
+      - --enable-libfontconfig
+      - --enable-libfreetype
+      - --enable-libopus
+      - --enable-libvorbis
+      - --enable-libvpx
+      - --enable-avresample
+      - --disable-libx264
+    cleanup:
+      - /share/ffmpeg
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-4.2.4.tar.xz
+        sha256: 0d5da81feba073ee78e0f18e0966bcaf91464ae75e18e9a0135186249e3d2a0b
+
+
+
+  - name: voicegen
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://gitlab.com/PersianGolf/voicegen.git
+        tag: "1.0.3"
+    post-install:
+      - install -Dm666 Integration-Stuff/desktop/voicegen.desktop /app/share/applications/io.gitlab.persiangolf.voicegen.desktop
```

Comment:
Note, since you are upstream: this is the kind of things that should be installed by your build system. Ideally it wouldn't need to be done here.

I'm mentionning this for a future release, it's not a blocker here at all.